internal/auth/telemetry_claim.go

```diff
@@ -51,3 +53,36 @@ func getTelemetryClaim(ctx context.Context) (*TelemetryClaim, error) {
 	}
 	return claim, nil
 }
+
+func ValidAttestationClaim(ctx context.Context, filter *model.AttestationFilter) bool {
+	claim, err := getTelemetryClaim(ctx)
+	if err != nil || claim.CloudEvents == nil {
+		return false
+	}
+
```

Comment:
It might be worth having a function that starts right here, and then that tests are really pure-logic-looking. Just a thought. I guess creating a context and stuffing a value into it isn't too bad.